dotnet/src/Connectors/Connectors.AI.OpenAI/OpenAIMemoryBuilderExtensions.cs

```diff
@@ -36,9 +36,9 @@ public static class OpenAIMemoryBuilderExtensions
         return builder.WithTextEmbeddingGeneration((loggerFactory, httpClient) =>
             new AzureOpenAITextEmbeddingGeneration(
                 deploymentName,
+                modelId,
```

Comment:
Could you please also change the ordering above in `WithAzureOpenAITextEmbeddingGeneration` method and XML comments? I think this extension method and constructor of `AzureOpenAITextEmbeddingGeneration` should be similar in terms of parameter ordering. At least, in both cases `modelId` should be after `deploymentName`.

Thank you!

Reply:
agreed

Reply:
@StuartMorrisHitachi Thanks! I added small changes in this commit as well: https://github.com/microsoft/semantic-kernel/pull/4012/commits/2540d995b2d77a448d1c2702e8db02af5d4fbfab